draft-ietf-lamps-kyber-certificates.md

```diff
@@ -357,6 +305,9 @@ prior example, the textual encoding defined in {{RFC7468}} is used:
   key pair and widest ability to import the key.
 </aside>
 
+{example-private} contains an example of an id-alg-ml-kem-768 private key
```

Comment:
Appendix says it's ML-KEM-512, here says 768.  Which one?  No actual examples to verify this against :)

Reply:
Yep 768!